index.js

```diff
@@ -335,7 +365,10 @@ function Physics (mcData, world) {
   }
 
   function getRenderedDepth (block) {
-    if (!block || block.type !== waterId) return -1
+    if (!block) return -1
+    if (block.type === bubblecolumnId) return 0
+    if (block.getProperties().waterlogged) return 0
```

Comment:
I've noticed in prismarine-viewer that water plants (sea grass, etc...) don't have the waterlogged property (as they are always underwater). Would be interesting to add them as well, but I don't know of a simple way to get all exceptions.

Reply:
they are all listed here https://minecraft.gamepedia.com/Waterlogging, we dont have any extractor for that page so ill just add them manually, as currently there are only 3 of them